casper/src/main/resources/PoS.rho

```diff
@@ -18,7 +18,7 @@ new PoS, rs(`rho:registry:insertSigned:ed25519`), uriOut in {
 
  for (@minimumBond, @maximumBond <- PoS) {
 
-    match "11112GofKy1v5mPWZjZEYBByvpZ4ahYJp6vFN1hgGCn7L543rFSpyA" {
+    match "1111jXe5SseJpLMkUbXQN9k9aNHifUVgKm1pew3fwydaahdtvRRo1" {
```

Comment:
It's hardcoded because the public key from which the address is derived is hardcoded. This could be parameterized if we unified `PoS.rho` with the `ProofOfStake` generated code.

Reply:
We should un-hardcode both of those places, probably before we progress with the next PR

Reply:
Having this hardcoded could be hiding quite a lot of dependencies which in turn might be making my understanding even more partial than it is already :wink: